src/app/modules/group/components/group-manager-list/group-manager-list.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, Input, OnChanges, OnDestroy, SimpleChanges } from '@angular/core';
 import { GetGroupManagersService, Manager } from '../../http-services/get-group-managers.service';
```

Comment:
> When permissions from "can_manage:memberships_and_group" to "can_manage:memberships" (or lower), the UI still let me reopen the edition popup and select managers for removal while the backend does not.

This part of the issue looks unimplemented. Steps to reproduce:
1. Go to any group, ie [Pixal](https://dev.algorea.org/branch/feature/manager-page-reducing-your-own-permissions/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
2. Create a subgroup (a club for instance)
3. Navigate to that new group
4. In the managers, edit your own permissions and set management level to "memberships"
5. Save

:arrow_right: Although the table is refreshed, the "Edit permissions" button is still displayed
:arrow_right: The "remove" checkboxes are still displayed